[2025-05-12T07:03:57.382+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-12T07:03:57.414+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monitor_PWU.monitor_model manual__2025-05-12T07:03:56.104193+00:00 [queued]>
[2025-05-12T07:03:57.425+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monitor_PWU.monitor_model manual__2025-05-12T07:03:56.104193+00:00 [queued]>
[2025-05-12T07:03:57.426+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-12T07:03:57.440+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): monitor_model> on 2025-05-12 07:03:56.104193+00:00
[2025-05-12T07:03:57.446+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monitor_PWU', 'monitor_model', 'manual__2025-05-12T07:03:56.104193+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/monitor_PWU.py', '--cfg-path', '/tmp/tmp2q4sb960']
[2025-05-12T07:03:57.450+0000] {standard_task_runner.py:91} INFO - Job 40: Subtask monitor_model
[2025-05-12T07:03:57.453+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=10777) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-12T07:03:57.454+0000] {standard_task_runner.py:63} INFO - Started process 10778 to run task
[2025-05-12T07:03:57.505+0000] {task_command.py:426} INFO - Running <TaskInstance: monitor_PWU.monitor_model manual__2025-05-12T07:03:56.104193+00:00 [running]> on host 8045d3f859a3
[2025-05-12T07:03:57.598+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monitor_PWU' AIRFLOW_CTX_TASK_ID='monitor_model' AIRFLOW_CTX_EXECUTION_DATE='2025-05-12T07:03:56.104193+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-12T07:03:56.104193+00:00'
[2025-05-12T07:03:57.599+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-12T07:03:57.624+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-12T07:03:57.625+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/monitor_PWU.py", line 22, in monitor_model
    raise Exception("Flask prediction failed.")
Exception: Flask prediction failed.
[2025-05-12T07:03:57.636+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=monitor_PWU, task_id=monitor_model, execution_date=20250512T070356, start_date=20250512T070357, end_date=20250512T070357
[2025-05-12T07:03:57.649+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 40 for task monitor_model (Flask prediction failed.; 10778)
[2025-05-12T07:03:57.668+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-05-12T07:03:57.691+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-12T07:03:57.696+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
